containers/opcua-device-gateway/Dockerfile

```diff
@@ -40,14 +40,13 @@ RUN case "${VERSION}" in \
     esac \
     && wget -O opcua-device-gateway.jar "$DOWNLOAD_URL"
 
-COPY application-tenant.yaml .
+
```

Comment:
Ideally deleting stuff should just deleting the line and not add a blank line in place. It makes for a cleaner diff and history...but probably not worth doing anything about in this PR (unless you really want to)